vncviewer/parameters.cxx

```diff
@@ -158,12 +158,14 @@ StringParameter display("display",
 StringParameter menuKey("MenuKey", "The key which brings up the popup menu",
                         "F8");
 
+
 BoolParameter fullscreenSystemKeys("FullscreenSystemKeys",
                                    "Pass special keys (like Alt+Tab) directly "
                                    "to the server when in full-screen mode.",
                                    true);
 
 #ifndef WIN32
+BoolParameter useSSH("UseSSH", "Create SSH tunnel", false);
```

Comment:
Why this new parameter? Isn't it sufficient to leave the via setting empty to disable the feature?

Reply:
I think you're right.  I'll remove this parameter and just check if the host is a blank string or not.